src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -22,6 +22,10 @@
     #?(:clj [cljs.analyzer :as ana])
     [com.fulcrologic.fulcro.algorithms.indexing :as indexing]))
 
+;; Class of the routing target component, available in the notifications fns
+;; (:will-enter, :route-cancelled, :will-leave)
```

Comment:
Convert this comment to a docstring pls.

Reply:
Also, why is this available in route-cancelled and will-leave?  Are you just saying that because it's true, or because you see a reason?  I don't like to make API guarantees that we don't have to.

Reply:
Yes, it is true. I mentioned previously you did not like the dynamic var also because it was only for that one use case. So I extended it to all the notification functions, see L47 and L86.

I will convert the comment to a docstring but the var is not intended for external use, I think. Would it then make sense to annotate it with [`^:no-doc`](https://github.com/cljdoc/cljdoc/blob/master/doc/userguide/for-library-authors.adoc#hiding-namespaces--vars-in-documentation)?

Reply:
We could go 2 ways. If it is only ever used by warnings, then perhaps it isn't a documented thing, and instead should be surrounded by `when goog.DEBUG` so it is elided completely in production, making it obvious it is just an add-on for warnings.